lib/game.rb

```diff
@@ -0,0 +1,50 @@
+class Game
+  include Concerns::Winable
+  attr_accessor :board, :player_1, :player_2
+
+  def initialize(player_1 = Human.new("X"), player_2 = Human.new("O"), board = Board.new)
+    @board = board
+    @player_1 = player_1
+    @player_2 = player_2
+  end
+
+  def current_player
+    board.turn_count % 2 == 0 ? player_1 : player_2
+  end
+
+  def over?
+    draw? || won?
+  end
+
+  def draw?
+    board.full? && !won?
+  end
+
+  def winner
+    board.cells[won?.first] if won?

```

Comment:
@azemoh Generally, one-line methods are great in ruby. I tend to stick to a couple of guidelines:

1.  Try to avoid any lines that get too long, usually ~80 characters (according to the [Ruby Styleguide](https://github.com/styleguide/ruby)). This can sometimes be tricky with long variable names, and I tend to err on the side of clarity rather than strict adherence to this "rule"

1.  Avoid single line methods that are too complex. This is something that is more subjective and you will get a better feel for the more you write in Ruby. Broadly speaking, one to two operations are fine, and three is borderline. But more basically, if you come back to a single line method that you wrote and you scratch your head for a second, it would probably be better to rewrite it for greater clarity.

Working in development teams that change a lot, I have come to value readability of code very highly.